pkg/server/server.go

```diff
@@ -110,7 +110,7 @@ func listenAndServe(ctx context.Context, clients *clients.Clients, handler http.
 			Service: &v1.ServiceReference{
 				Namespace: namespace,
 				Name:      serviceName,
-				Path:      &validationPath,
+				Path:      &mutationPath,
```

Comment:
@thedadams Can you open a new pr with just this change so we can get it merged ASAP

Reply:
Done; https://github.com/rancher/webhook/pull/39